samples/testing-frameworks/appium/client-side/ruby/bitbar_webdriver_ios.rb

```diff
@@ -71,9 +71,8 @@ def upload_application(file_path, bitbar_api_key)
       log ("Upload application #{bitbar_app_file}")
       upload_application(bitbar_app_file, bitbar_api_key)
       log ("Uploaded file id #{@bitbar_app}")
```

Comment:
```suggestion
      log ("Uploaded file id #{@bitbar_app_id}")
```

It should be also changed in line 65 and similarly i all the other test files